run_dir/static/css/index.css

```diff
@@ -0,0 +1,17 @@
+#project_quotas .plot-column {
+	height: 60px;
+	padding: 0;
+}
+
+tr td:first-child {
```

Comment:
I know I said to use `td:first-child` instead of class names, but this is now _too_ general :smile: - this will make the first cell of every table row on the entire website 150px wide. Instead you want `#project_quotas tr td:first-child`